test/liveValidation/swaggers/specification/mediaservices/resource-manager/Microsoft.Media/2019-05-01-preview/Common.json

```diff
@@ -0,0 +1,80 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Media Services",
+    "version": "2018-07-01"
```

Comment:
Should this version be same as folder name?

Reply:
No, it is to create an issue of multiple operations in the same API version and this is actually a real case.